docs/jwst/source_catalog/main.rst

```diff
@@ -232,6 +232,11 @@ columns (assuming the default encircled energies of 30, 50, and 70):
 |                        | minimal bounding box of the source                 |
 +------------------------+----------------------------------------------------+
 
+Note that pixel coordinates are 0 indexed, matching the Python 0-based
+indexing. That means the first pixel is considered pixel ``0`` and pixel
+coordinate ``0`` is the center of that pixel.
```

Comment:
I think "first pixel is considered pixel ``0``" is a little confusing. I think the first sentence is basically enough. Or leave only this from the second sentence:

```
That means the coordinate first pixel's center is ``0``.
```

but it kind of repeats the idea from the first sentence.

Reply:
OK, I removed the the first part of the second sentence.